.github/workflows/build.yml

```diff
@@ -129,7 +129,7 @@ jobs:
           CFLAGS: "/UNDEBUG"
           CXXFLAGS: "/UNDEBUG"
       - name: Run tests
-        run: ctest -C Release --output-on-failure --parallel %NUMBER_OF_PROCESSORS%
```

Comment:
if this is run on windows, %NUMBER_OF_PROCESSORS% is the normal way to get to environment variables

Reply:
Yup, and this currently fails now and then, e.g. [here](https://github.com/google/XNNPACK/actions/runs/8448281534/job/23140060147#step:4:5).